Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -57,6 +58,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed Input Actions Editor losing reference to current ControlScheme upon entering Play Mode [ISXB-770](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-770).
 - Fixed Save shortcut (ctrl/cmd + S by default) not saving changes in Input Actions Editor windows. [ISXB-659](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-659).
 - Fixed headers in InputActionEditor windows becoming squashed when there is a large number of Action Maps/Actions.
+- Fixed duplication of project wide input actions when loading/unloading scenes [ISX-1902]
```

Comment:
Remove ISX-1902 from this since its an internal ticket. We only link public bug tickets ISXB and for them use issue tracker links.

Reply:
@AlexTyrer This one needs to be fixed before merge, but approved to not block this PR from landing.

Reply:
OK - I saw there are other some internal ISX- issues in the file (with no links) so assumed that was the thing to do.

Removed in 1da1021